harness/wellKnownIntrinsicObjects.js

```diff
@@ -140,8 +150,12 @@ const WellKnownIntrinsicObjects = [
     source: 'isNaN',
   },
   {
-    name: '%IteratorPrototype%',
```

Comment:
this intrinsic name will still be in the spec, so we should probably still have it in this list?

Reply:
It is no longer there as far as I can tell. It seems to have been replaced by %Iterator.prototype% which means you can get it in this case with `getWellKnownIntrinsic('%Iterator%').prototype`

Reply:
hmm, that change was my PR, and the intention was to leave intact any pre-existing intrinsics, so that might be a spec bug I’ll have to fix.